snare/tests/test_parse_tanner_response.py

```diff
@@ -73,6 +105,54 @@ def test_parse_type_two(self):
         expected_result = [self.expected_content, self.content_type, {}, 200]
         self.assertCountEqual(real_result, expected_result)
 
+    def test_parse_type_two_with_headers(self):
+
+        self.detection = {
+            "type": 2,
+            "payload": {
+                "page": "",
+                "value": "test.png",
+                "headers": {
+                    "content-type": "multipart/form-data"
+                }
+            }
+        }
+        self.expected_content = b'test.png'
+        self.content_type = 'image/png'
+        self.expected_header = {"content-type": "multipart/form-data"}
+
+        async def test():
+            (self.res1, self.res2,
+             self.res3, self.res4) = await self.handler.parse_tanner_response(self.requested_name, self.detection)
+
+        self.loop.run_until_complete(test())
+        real_result = [self.res1, self.res2, self.res3, self.res4]
+        expected_result = [self.expected_content, self.content_type, self.expected_header, 200]
+        self.assertCountEqual(real_result, expected_result)
+
+    def test_parse_type_two_error(self):
+
+        self.detection = {
+            "type": 2,
+            "payload": {
+                "page": "/index.html",
+                "value": "test"
+            }
+        }
+        meta_content = {"/index.html": {}}
+        self.handler = TannerHandler(self.args, meta_content, self.uuid)
+        self.expected_content = b'<html><body><div>test</div></body></html>'
+        self.content_type = r'text\html'
```

Comment:
I think it should be 
```suggestion
        self.content_type = r'text/html'
```
Right?
And can you first make a separate PR to fix this https://github.com/mushorg/snare/blob/c04984110c15d8fbe031a77d51af7f17a423ca5b/snare/tanner_handler.py#L109

Reply:
Roger!

Reply:
PR submitted.

Reply:
Fix it.